subfun/BIDS_RFX.m

```diff
@@ -76,17 +76,17 @@ function BIDS_RFX(action, mmFunctionalSmoothing, mmConSmoothing, opt)
                 % FFX Directory
                 ffxDir = getFFXdir(subNumber, mmFunctionalSmoothing, opt);
                 conImg = spm_select('FPlist', ffxDir, '^con*.*nii$');
-                matlabbatch{1}.spm.spatial.smooth.data = cellstr(conImg);
+                matlabbatch{counter}.spm.spatial.smooth.data = cellstr(conImg);
+
+                % Define how much smoothing is required
+                matlabbatch{counter}.spm.spatial.smooth.fwhm = [mmConSmoothing mmConSmoothing mmConSmoothing];
+                matlabbatch{counter}.spm.spatial.smooth.dtype = 0;
+                matlabbatch{counter}.spm.spatial.smooth.prefix = [...
+                spm_get_defaults('smooth.prefix'), num2str(mmConSmoothing)];
 
             end
         end
 
-        % Define how much smoothing is required
-        matlabbatch{1}.spm.spatial.smooth.fwhm = [mmConSmoothing mmConSmoothing mmConSmoothing];
-        matlabbatch{1}.spm.spatial.smooth.dtype = 0;
-        matlabbatch{1}.spm.spatial.smooth.prefix = [...
-            spm_get_defaults('smooth.prefix'), num2str(mmConSmoothing)];
-
```

Comment:
this is removed for the matlabbatch{1} issue, and added above inside the subjects loop